src/App.css

```diff
@@ -55,20 +58,212 @@ input {
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
+  font-size: x-large;
   letter-spacing: -0.18px;
   color: #888;
   border-top: solid 1px #e8e8e8;
 }
 
 .stylebook {
   list-style: none;
-  flex-direction: column;
+  display: flex;
+  flex-direction: row;
+  border: 1px solid black;
+  background: white;
+  padding-top: 1em;
+  align-items: center;
+  justify-content: space-between;
 }
 
 a {
   font-size: medium;
+  text-decoration: none !important;
+  color: black !important;
```

Comment:
- [ ] I advise that you do not use !important until it is absolutely necessary. In this case, you can do this instead:

``` 
.linkstyles > a {
  font-size: medium;
  text-decoration: none;
  color: black;
}
```

Reply:
ok im going to change that